packages/repository/test/unit/type/type.ts

```diff
@@ -494,4 +494,83 @@ describe('types', () => {
       expect(unionType.serialize(undefined)).undefined();
     });
   });
+
+  describe('model', () => {
+    class Address extends types.ValueObject {
+      street: string;
+      city: string;
+      zipCode: string;
+    }
+
+    class Customer extends types.Entity {
+      id: string;
+      email: string;
+      address: Address;
+
+      constructor(data?: types.AnyObject) {
+        super();
+        if (data != null) {
+          this.id = data.id;
+          this.email = data.email;
+          this.address = data.address;
+        }
+      }
+    }
+
+    const modelType = new types.ModelType(Customer);
+    it('checks isInstance', () => {
+      expect(modelType.isInstance('str')).to.be.false();
+      expect(modelType.isInstance(null)).to.be.true();
+      expect(modelType.isInstance(undefined)).to.be.true();
+      expect(modelType.isInstance(NaN)).to.be.false();
+      expect(modelType.isInstance(true)).to.be.false();
+      expect(modelType.isInstance({id: 'c1'})).to.be.false();
+      expect(modelType.isInstance([1, 2])).to.be.false();
+      expect(modelType.isInstance(1)).to.be.false();
+      expect(modelType.isInstance(1.1)).to.be.false();
+      expect(modelType.isInstance(new Customer())).to.be.true();
```

Comment:
One assert per test please.

Can you perhaps rewrite these tests a parameterized tests?

```ts
describe('isInstance', () => {
  const TEST_CASES = [
    ['str', false],
    [null, true],
    // ...
  ];

  TEST_CASES.forEach(([input, output]) => {
    it(`returns ${output} for ${input}`, () => {
      expect(modelType.isInstance(input)).to.equal(output);
    });
  });
});
```